ruby_programming/object_oriented_programming_basics/lesson_oop.md

```diff
@@ -103,3 +60,13 @@ This section contains helpful links to other content. It isn't required, so cons
 * Now you're ready to read through [Zetcode's OOP section](http://zetcode.com/lang/rubytutorial/oop/).
 * Read through [Zetcode's second OOP section](http://zetcode.com/lang/rubytutorial/oop2/) until they start talking about exceptions (~80% of the way down).
 
+### Knowledge Check
+
+This section contains questions for you to check your understanding of this lesson. If you’re having trouble answering the questions below on your own, review the material above to find the answer.
+
+- <a class="knowledge-check-link" href="https://www.eriktrautman.com/posts/ruby-explained-classes">What is a class?</a>
+- <a class="knowledge-check-link" href="https://www.eriktrautman.com/posts/ruby-explained-classes">When should you use a class?</a>
```

Comment:
Are these questions directly answered in the linked resource? I'm not seeing them but I 100% could just be missing them.

Reply:
Not directly answered. Eriks post indirectly answers it. I'll update the links to a more directly answered resource.